templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -242,34 +242,10 @@ contents:
       # Get the new connection uuid
       new_conn=$(nmcli -g connection.uuid conn show "$bridge_interface_name")
 
-      # Setup an exit trap to restore any modifications going further
-      handle_exit_error() {
```

Comment:
Looks you are just consolidating logic into better named functions in this commit. Would be nice to list in the commit message what you are changing.